app/views/make-meeting.jade

```diff
@@ -34,6 +35,7 @@ div.meeting
           i.fa.fa-times-circle.warn
   div.col-lg-12.when
     h2 When is it
+      span.instructions Here's the chunks of times that work for all attendees. Choose some and we'll pick the best one
```

Comment:
Here are the chunks of time that work for everyone.  Choose one, or a few and Tandem will do the rest.